include/caf/leds.rst

```diff
@@ -18,19 +18,18 @@ To use the module, you must enable the following Kconfig options:
 * :option:`CONFIG_CAF_LEDS` - This option enables the LEDs module.
 * :option:`CONFIG_PWM` - This option enables Zephyr's :ref:`zephyr:pwm_api` driver, which is used for setting the LED color (that is, the brightness of the diodes).
   Setting this option requires you to define PWM ports, as described in the following section.
+* :option:`CONFIG_LED` - This option enables the LED driver API.
+* :option:`CONFIG_LED_PWM` - This option enabled the LED PWM driver.
+  Setting this option requires you to define PWM LEDs, as described in the following section.
 
 The following Kconfig options are available for this module:
 
-* :option:`CONFIG_CAF_LEDS_DEF_PATH`
 * :option:`CONFIG_CAF_LEDS_PM_EVENTS`
-* :option:`CONFIG_CAF_LEDS_COUNT`
-* :option:`CONFIG_CAF_LEDS_COLOR_COUNT`
-* :option:`CONFIG_CAF_LEDS_BRIGHTNESS_MAX`
 
-Defining PWM ports
-==================
+Preparing DTS configuration
+===========================
 
-To define PWM ports used by the application, complete the following steps:
+To use LEDs driven by PWM, complete the following steps:
```

Comment:
This suggests you can also use other LEDs, not driven by PWM. Is this the case?

Reply:
in general yes, we don't support it (yet)

Should it be "To use LEDs through the LED PWM driver"? "To use LEDs" is too generic in my opinion.